Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -248,9 +248,10 @@ public class PBXProjGenerator {
                 addObject(targetDependency)
                 dependencies.append(targetDependency.reference)
 
-                // don't bother linking a target dependency
-                // let dependencyBuildFile = targetBuildFileReferences[dependencyTargetName]!
-                // targetFrameworkBuildFiles.append(dependencyBuildFile)
+                if dependencyTarget.type.isLibrary || dependencyTarget.type.isFramework {
+                    let dependencyBuildFile = targetBuildFileReferences[dependencyTargetName]!
+                    targetFrameworkBuildFiles.append(dependencyBuildFile)
```

Comment:
Hmm, this seems to not be the type that Xcode is expecting.

Reply:
Oops, I was testing with a project without a bundle identifier, which really confuses Xcode